hydrus/data/alembic.ini

```diff
@@ -0,0 +1,75 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = migrations
+
+# template used to generate migration files
+# file_template = %%(rev)s_%%(slug)s
+
+# timezone to use when rendering the date
+# within the migration file as well as the filename.
+# string value is passed to dateutil.tz.gettz()
+# leave blank for localtime
+# timezone =
+
+# max length of characters to apply to the
+# "slug" field
+# truncate_slug_length = 40
+
+# set to 'true' to run the environment during
+# the 'revision' command, regardless of autogenerate
+# revision_environment = false
+
+# set to 'true' to allow .pyc and .pyo files without
+# a source .py file to be detected as revisions in the
+# versions/ directory
+# sourceless = false
+
+# version location specification; this defaults
+# to alembic/versions.  When using multiple version
+# directories, initial revisions must be specified with --version-path
+# version_locations = %(here)s/bar %(here)s/bat alembic/versions
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+# hydrus users please change this line with the address to your Database
+sqlalchemy.url = sqlite:///database.db
```

Comment:
Can we change this to take the URL from `os.environ`? The same as we did in `conf.py`.

This might be helpful: https://stackoverflow.com/questions/37890284/ini-file-load-environment-variable

Reply:
Sure, I will take a look. I can also just import DB_URL from conf.py, makes sense?

Reply:
Sure, I will take a look. I can also just import DB_URL from conf.py.

Reply:
It's done, it works life conf.py and also rewrote the readme.